docs/sources/next/javascript-api/k6-experimental/csv/parse.md

```diff
@@ -0,0 +1,82 @@
+---
+title: 'parse'
+description: 'parse a CSV file into a SharedArray'
+weight: 20
+---
+
+# parse
```

Comment:
```suggestion
title: 'parse( file, [options] )'
description: 'parse a CSV file into a SharedArray'
weight: 20
---

# parse( file, [options] )
```

I think there's some inconsistency in the docs that I need to fix, but the majority of the functions that take parameters have them in the page title. Does this look accurate to you?

Reply:
There definitely is, as I usually have no clue how to name stuff. Also, the name contains the function's parameters, but not the file name. This is not a problem, but I'd love some guidelines on this.

Let me know if you want me to help with adjusting the existing 👍🏻 Maybe we can add it to our plate for our recurring sessions too 🙇🏻

Reply:
Yeah, that's a good idea. It would be nice to if we could create some sorta template for these files as well, I think that would help with consistency around the tables and sections. I'll create an issue for it.